README.md

```diff
@@ -1,18 +1,41 @@
 # Azure Exporter for Scrapy
+scrapy_azure_exporter is a Python package which can be used in integration with scrapy for exporting feeds to Azure Storage.
+## Description:
+The scrapy_azure_exporter takes inspiration from the built-in feed exporters in scrapy. The built-in exporters doesn't have support for Azure yet.
+So this scrapy_azure_exporter can be used whenever when we need the feeds to be exported to Azure Blob storage.
 
-## Requirements:
-- Python 3.8
-- Pipenv
+## Getting started:
+### Installation:
+```bash
+> pip install git+https://github.com/scrapy-plugins/scrapy-feedexporter-azure-storage
+```
+### Usage:
+* Add scrapy_azure_exporter to spider's settings.
+```python
+"FEED_STORAGES" : {'azure': 'scrapy_azure_exporter.AzureFeedStorage'}
+```
+* Provide authentication for Azure account via any of the three methods specified in the configuration section.
+```python 
+For example, "AZURE_CONNECTION_STRING": "Connection string for the Azure account"
+```
```

Comment:
What if we describe here the authentication parameters, and rename the Configuration section as Additional options, and cover only things not covered here?